docs/infra/treatment-service.md

```diff
@@ -0,0 +1,132 @@
+# Treatment Service Deployment
+
+## Overview
+XP's Treatment Service is currently offered in two different versions:
+
+- a central Treatment Service, *deployed as a single deployment, to serve treatments to multiple 
+  clients/users with multiple projects*
+- a Treatment Service plugin, *deployed as a Turing Router experiment engine plugin, to serve treatments to a single 
+  client/user with a single project*
+
+## Central Treatment Service
+
+### Requirements
+- Configuration
+- Google Cloud Provider (GCP) service account
+
+### Configuration
+The central Treatment Service deployment uses a variety of configuration that can be stored in one or multiple 
+configuration files (`.yaml`) files. Some of such examples can be found here:
+
+- https://github.com/caraml-dev/xp/treatment-service/config/example.yaml
+- https://github.com/caraml-dev/xp/treatment-service/testdata/config1.yaml
+- https://github.com/caraml-dev/xp/treatment-service/testdata/config2.yaml
+
+As can be seen from the various examples, there are a number of values that need to be set, though not all of them 
+are necessarily required. Some of these values, when left undefined/empty, will be automatically initialised with 
+certain default values (see https://github.com/caraml-dev/xp/blob/main/treatment-service/config/config.go#L22).
+
+### Google Cloud Provider (GCP) Service Account
+[Google Cloud Pub/Sub](https://cloud.google.com/pubsub/docs/overview) is used to allow the Treatment Service to 
```

Comment:
`is used to` -> `is required for Treatment Service...`. It's a hard dependency now.

Reply:
Good point. Corrected that.